spec/views/notifications/index.html.erb_spec.rb

```diff
@@ -15,26 +14,26 @@
   let(:patch_note_2) { create(:patch_note, note: "Patch Note B", patch_note_type: patch_note_type_b) }
 
   before do
-    assign(:notifications, Noticed::Notification.all)
+    assign(:notifications, Noticed::Notification.all.newest_first)
     assign(:patch_notes, PatchNote.all)
     assign(:deploy_time, deploy_time)
   end
 
   context "when there is a deploy date" do
-    let(:deploy_time) { base_time - 2.days }
+    let(:deploy_time) { 2.days.ago }
 
     before do
       Health.instance.update_attribute(:latest_deploy_time, deploy_time)
     end
 
     context "when there are notifications" do
       before do
-        notification_1_hour_ago.update_attribute(:created_at, base_time - 1.hour)
-        notification_1_day_ago.update_attribute(:created_at, base_time - 1.day)
-        notification_2_days_ago.update_attribute(:created_at, base_time - 2.days)
-        notification_3_days_ago.update_attribute(:created_at, base_time - 3.days)
+        notification_1_hour_ago.update!(created_at: 1.hour.ago)
+        notification_1_day_ago.update!(created_at: 1.day.ago)
+        notification_2_days_ago.update!(created_at: 2.days.ago)
+        notification_3_days_ago.update!(created_at: 3.days.ago)
 
-        patch_note_1.update_attribute(:patch_note_group, patch_note_group_all_users)
+        patch_note_1.update!(patch_note_group: patch_note_group_all_users)
```

Comment:
these are importantly different and I suspect defeat the point of the test.

Reply:
`update!` will fail if validation fails, `udpate_attribute` skips validation.  This is in a before block, so it's setup for the actual tests... so we should want it to fail if the setup is 'invalid'.

That was my thinking, anyway. Probably had rubocop-rspec running when I did this.